test/testdrive/introspection-sources.td

```diff
@@ -295,7 +295,7 @@ true true true
 
 > INSERT INTO t4 SELECT generate_series(1, 1000)
 
-> SELECT records >= 1000 AND records <= 1001, batches > 0, size > 64*1000 AND size < 2*64*1000, capacity > 64*1000, allocations > 0 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name LIKE '%ii_t4'
+> SELECT records >= 1000 AND records <= 1001, batches > 0, size > 48*1000 AND size < 2*48*1000, capacity > 48*1000, allocations > 0 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name LIKE '%ii_t4'
```

Comment:
Same as above.

Reply:
I think these values are correct now, after https://github.com/MaterializeInc/materialize/pull/23033 merged? Let me know if they're not and I can update them